### PR TITLE
chore(docs): add examples for ucloud provider (ts/python/java/c#/go)

### PR DESCRIPTION
**Pull request by yufeiminds**
Add examples for [UCloud](https://www.ucloud.cn/) provider.

#### Languages

- [x] TypeScript
- [x] Python
- [x] Java
- [x] C#
- [x] Go  

#### Additional Infomation

- [UCloud Provider Docs](https://registry.terraform.io/providers/ucloud/ucloud/latest/docs)

### CODE REVIEW

**Review by DanielMSchmidt — APPROVED** [review]
Awesome, thanks a ton for sending in a PR 👍

File: examples/typescript/ucloud/main.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { Construct } from 'constructs';
+import { App, TerraformStack } from 'cdktf';
+import { UcloudProvider, Instance, DataUcloudImages } from './.gen/providers/ucloud'
+
+class MyStack extends TerraformStack {
+  constructor(scope: Construct, name: string) {
+    super(scope, name);
+
+    new UcloudProvider(this, 'UCloud', {
+      region: 'cn-bj2',
```
Comment (DanielMSchmidt) [inline examples/typescript/ucloud/main.ts:10]: It seems like projectId is missing here, judging the failing test

Comment (yufeiminds) [reply examples/typescript/ucloud/main.ts:10]: Thanks for your reply, provider load the environment variable when I tested it on local, and I just fix it. How can I retry this test case manually?

Comment (DanielMSchmidt) [reply examples/typescript/ucloud/main.ts:10]: I think your last commit made TS happy 🙌

**Comment by jsteinich** [comment]
Running on Windows currently requires installing all of the tools for each language you are testing with.

Looks like you are running correctly within the container. Could have been a temporary issue or perhaps some network configuration issue on your machine.

**Comment by yufeiminds** [comment]
Hi @DanielMSchmidt,

Thanks for @jsteinich, the integration test `yarn examples:integration:typescript` has been successful on my local, but I don't have a windows server to reproduce the `integration:windows:java` failure.

In this [log details](https://github.com/hashicorp/terraform-cdk/pull/790/checks?check_run_id=2906124027), this is a timeout error, could you approval to continue the workflow, please. thank you ~ 😁

**Comment by yufeiminds** [comment]
I had passed the `yarn build`, and `prettier --check .`.

But after `yarn bootstrap-plugin-cache` and `yarn test`, it raised:

```
FAIL test/config.test.ts
  ● parseConfig › modules › parses module for local module for module generator

    expect(received).toMatch(expected)

    Expected substring: "terraform-cdk/packages/cdktf-cli/foo"
    Received string:    "file:///home/packages/cdktf-cli/foo"

      133 |       };
      134 |       const parsed: any = parseConfig(JSON.stringify(input));
    > 135 |       expect(parsed.terraformModules[0].localSource).toMatch(
          |                                                      ^
      136 |         "terraform-cdk/packages/cdktf-cli/foo"
      137 |       );
      138 |     });

      at Object.it (test/config.test.ts:135:54)
```

Can you help me to merge this pull request? What should I do for the next step? Thanks a lot 😄  @DanielMSchmidt

**Comment by yufeiminds** [comment]
Thanks to @ansgarm. I have passed the `yarn test` stage. 😄

**Comment by yufeiminds** [comment]
Hello, thanks for your help. Can this PR be merged? @DanielMSchmidt @jsteinich  @ansgarm

**Review by ansgarm — APPROVED** [review]

**Pull request merged**